domains/nishanth.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Nishanth-S-A",
+        "email": "nishanthselvakumar05@gmail.com"
+    },
+    "record": {
+        "A": ["192.168.1.111"]
```

Comment:
I cannot reach your site
"192.168.1.111 took too long to respond."

Reply:
Sorry, I was trying it out. Is there a way to delete the request?